README.md

```diff
@@ -1,10 +1,10 @@
 ![XYZ Hub](xyz.svg)
----
+
```

Comment:
Could we keep the horizontal line after the logo?

Reply:
Sorry, reverted the change.